app/routes/docs/schema-browser.tsx

```diff
@@ -0,0 +1,131 @@
+/*!
+ * Copyright © 2022 United States Government as represented by the Administrator
+ * of the National Aeronautics and Space Administration. No copyright is claimed
+ * in the United States under Title 17, U.S. Code. All Other Rights Reserved.
+ *
+ * SPDX-License-Identifier: NASA-1.3
+ */
+import { Link, NavLink, Outlet, useLoaderData } from '@remix-run/react'
+import { Icon } from '@trussworks/react-uswds'
+import dirTree from 'directory-tree'
+import { useState } from 'react'
+
+import { SideNav, SideNavSub } from '~/components/SideNav'
+import { feature } from '~/lib/env.server'
+
+// Schema treeItem
+type SchemaTreeItem = {
+  name: string
+  path: string
+  children?: SchemaTreeItem[]
+}
+
+export async function loader() {
+  if (!feature('SCHEMA')) throw new Response(null, { status: 404 })
+  const localDataTree = dirTree(
+    '../../node_modules/@nasa-gcn/schema/gcn/notices'
+  )
+
+  return { localDataTree }
+}
+
+export default function Schema() {
+  const { localDataTree } = useLoaderData<typeof loader>()
+
+  const items: React.ReactNode[] = ([localDataTree] as SchemaTreeItem[])
+    .filter((x) => !x.name.includes('.example.json'))
+    .map(RenderSchemaTreeItem)
```

Comment:
You can do this filtering in the loader to reduce the amount of data that is sent to the browser.